mongodbatlas.go

```diff
@@ -126,6 +128,56 @@ func (m *MongoDBAtlas) NewUser(ctx context.Context, req dbplugin.NewUserRequest)
 		return dbplugin.NewUserResponse{}, err
 	}
 
+	clusters, _, err := client.Clusters.List(ctx, m.ProjectID, &mongodbatlas.ListOptions{})
+	if err != nil {
+		return dbplugin.NewUserResponse{}, fmt.Errorf("roles array is required in creation statement")
+	}
+
+	clusterList := []string{}
+	for _, c := range clusters {
+		clusterList = append(clusterList, c.Name)
+	}
+
+	var wg sync.WaitGroup
+	result := make(chan error, 10)
+	wg.Add(len(clusterList))
+
+	for _, c := range clusterList {
+
+		go func(clustername string) {
+
+			operation := func() error {
+
+				status, _, err := client.Clusters.Status(context.Background(), m.ProjectID, clustername)
```

Comment:
any reason not to reuse the context from outside the loop?

Reply:
Nope, will fix this